test/functional/searches_controller_test.rb

```diff
@@ -9,6 +9,15 @@ class SearchesControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test "new search page when query present at /search?q=" do
```

Comment:
Oh cool - does this still support searching with the route `/search/QUERY`?

Thanks @Mujadded !!

Reply:
Just checking in - I think this could with just a route addition, no?